concordia/admin/views.py

```diff
@@ -112,6 +112,20 @@ def redownload_images_view(request):
     return render(request, "admin/redownload_images.html", context)
 
 
+@never_cache
+@staff_member_required
+@permission_required("concordia.add_campaign")
+@permission_required("concordia.change_campaign")
+@permission_required("concordia.add_project")
+@permission_required("concordia.change_project")
+@permission_required("concordia.add_item")
+@permission_required("concordia.change_item")
+def celery_task_review(request):
```

Comment:
This function is named in a way which suggests that it's using Celery but it doesn't appear to do anything other than display the messages pending for the active user. That will also mean that this will be empty most of the time since messages only display once so you would only see it if you went to this page without hitting any other page.